src/lib/wallet/IndyWallet.ts

```diff
@@ -68,6 +63,20 @@ export class IndyWallet implements Wallet {
     return this.indy.createAndStoreMyDid(this.wh, didConfig || {});
   }
 
+  async createCredDef(
```

Comment:
This may be really nit-picky but I think we should either name it `createCredentialDef()` as indy does with `issuerCreateAndStoreCredentialDef` or use the full name `createCredentialDefinition`